modules/phase_field/src/action/MatVecRealGradAuxKernelAction.C

```diff
@@ -31,38 +34,55 @@ MatVecRealGradAuxKernelAction::MatVecRealGradAuxKernelAction(const InputParamete
 void
 MatVecRealGradAuxKernelAction::act()
 {
-  std::vector<unsigned int> _op_num = getParam<std::vector<unsigned int> >("op_num");
+  unsigned int _op_num = getParam<unsigned int>("op_num");
   std::vector<std::string> _var_name_base = getParam<std::vector<std::string> >("var_name_base");
```

Comment:
This shadows the member, which is **not** a vector type. Please remove the member and remove the leading `_` here.